src/config/custom.ts

```diff
@@ -0,0 +1,15 @@
+export const CUSTOM_CONFIG = {
```

Comment:
@arjun289 could you remove this file. It is not required.

Reply:
sure :+1:

Reply:
yes remove it, we will pushed custom folder. PR review from visula code 😀